src/wp-includes/formatting.php

```diff
@@ -3162,7 +3176,10 @@ function wp_rel_callback( $matches, $rel ) {
 		}
 		$text = trim( $html );
 	}
-	return "<a $text rel=\"" . esc_attr( $rel ) . '">';
+
+	$rel_attr = $rel ? ' rel="' . esc_attr( $rel ) . '"' : '';
+
+	return "<a {$text}{$rel_attr}>";
```

Comment:
hey @aaronjorbin this change will result in a double space when the rel attribute is present and an unnecessary space before the closing bracket when the attr is not present  since the space is included in the `$rel_attr` var on line 3180

Reply:
Thanks.  I fixed that up